compilers/concrete-compiler/compiler/include/concretelang/Dialect/FHELinalg/IR/FHELinalgOps.td

```diff
@@ -1517,5 +1517,59 @@ def FHELinalg_FancyAssignOp : FHELinalg_Op<"fancy_assign", [Pure]> {
     let hasVerifier = 1;
 }
 
+def FHELinalg_BroadcastOp: FHELinalg_Op<"broadcast", [Pure, ConstantNoise]> {
+    let summary = "Broadcasts a tensor to a shape.";
+    let description = [{
+        Examples:
```

Comment:
A short summary in one or two sentences of how the elements of the resulting tensor are related to the operand would be good. Right now, you need to go through the examples and extract that information yourself.

Reply:
I'll add a link to the [broadcasting documentation](https://numpy.org/doc/stable/user/basics.broadcasting.html#general-broadcasting-rules) of NumPy, which we mimic exactly.

Reply:
If it's possible to summarize this in a one or two sentences, I think this would be good in addition to the link.

Reply:
Sure.